core/src/integration-test/java/com/scalar/db/storage/dynamo/DynamoEnv.java

```diff
@@ -45,7 +45,8 @@ public static Properties getProperties(String testName) {
 
     // Add testName as a metadata namespace suffix
     props.setProperty(
-        DynamoConfig.METADATA_NAMESPACE, DynamoAdmin.METADATA_NAMESPACE + "_" + testName);
+        DatabaseConfig.SYSTEM_NAMESPACE_NAME,
+        DatabaseConfig.DEFAULT_NAMESPACE_NAME + "_" + testName);
```

Comment:
```suggestion
        DatabaseConfig.DEFAULT_SYSTEM_NAMESPACE_NAME + "_" + testName);
```
I guess you didn't intend to set `scalar.db.system_namespace_name` property to `scalar.db.default_namespace_name_${testName}`.

Reply:
Good catch! Fixed in efd4c49d99676ff5d150c9bf74850c50b22c6a9c. Thanks.